package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lint:fix": "prettier --write ."
```

Comment:
why need to use others packages when you can fix it with command?

Reply:
I tried but I donno why the commands that you gave me aren't working